package.json

```diff
@@ -16,12 +16,14 @@
     "@emotion/react": "^11.11.4",
     "@emotion/styled": "^11.11.5",
     "@fontsource/poppins": "^5.0.13",
+    "@fontsource/roboto": "^5.0.12",
     "@mui/icons-material": "^5.15.15",
     "@mui/material": "^5.15.15",
+    "@mui/styled-engine-sc": "^6.0.0-alpha.18",
     "firebase": "^10.10.0",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
-    "react-router-dom": "^6.13.0"
+    "styled-components": "^6.1.8"
```

Comment:
I am not sure we need `styled-components` package. Remove it if it is not necessary.

Reply:
Removed the dependency. And fixed issue with CI/CD deployment